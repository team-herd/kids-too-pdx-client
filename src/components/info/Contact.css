.Contact {
  padding: 100px;
  text-align: center;
}
image {
  height: 200%;
}