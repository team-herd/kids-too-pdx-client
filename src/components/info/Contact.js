import React from 'react';
import styles from './Contact.css';

function Contact() {
  return (
    <>
    <section className={styles.Contact}>
      <h1>Kids Too Inc.</h1>
      <p><strong>Address:</strong> 200 Happy Place, Portland OR 97003</p>
      <p><strong>Phone: </strong> (503) 200 400</p>
      <p><strong>Email:</strong> kidstoo@gmail.com</p>
    </section>
    </>
  );
}

export default Contact;
