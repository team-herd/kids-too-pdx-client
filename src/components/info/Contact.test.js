import React from 'react';
import renderer from 'react-test-renderer';
import Contact from './Contact';

describe('Contact', () => {
  it('snapshots Contact', () => {
    const tree = renderer.create(
      <Contact />
    );

    expect(tree).toMatchSnapshot();
  });
});
