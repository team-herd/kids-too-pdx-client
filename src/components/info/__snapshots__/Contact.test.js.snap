// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact snapshots Contact 1`] = `
<section
  className="Contact"
>
  <h1>
    Kids Too Inc.
  </h1>
  <p>
    <strong>
      Address:
    </strong>
     200 Happy Place, Portland OR 97003
  </p>
  <p>
    <strong>
      Phone: 
    </strong>
     (503) 200 400
  </p>
  <p>
    <strong>
      Email:
    </strong>
     kidstoo@gmail.com
  </p>
</section>
`;
